app/web/features/profile/locales/en.json

```
@@ -127,37 +128,61 @@
   "report_reason": "Reason",
   "report_user": "Report this user",
   "leave_reference": {
+    "add_a_reference": "Add a reference",
     "appropriate_behavior": "Appropriate Behavior",
-    "appropriate_explanation": "Awesome –– We hope you had a great time! To help keep our community safe, we want to ask about your interaction with your fellow Coucher.",
+    "appropriate_explanation": "References keep users accountable and reward great experiences. Thank you for doing your part to help keep the community safe, trustworthy, and welcoming for everyone! ",
     "appropriate_question": "Did you feel safe with this person's behavior?",
+    "by_writing_thoughtful": "By writing a thoughtful reference, you are doing your part to keep the community a safe, trustworthy, and welcoming place for everyone!",
     "coucher_was_appropriate": "Yes, this person's behavior was appropriate.",
     "coucher_was_not_appropriate": "No, this person's behavior was not appropriate.",
     "contact_text": "If you have any questions or wish to provide additional information, please don't hesitate to <1>contact us here.</1>",
+    "did_stay_explanation": "You <strong>will not be able to write a reference</strong> if you indicate you did not meet up.\n The other member <strong>will not be notified</strong> and they can still leave a reference. If you need to report inappropriate or unwanted behavior, please report this user.",
```

Since 90+% of people should click `yes` when asked about `DidStay`, we don't want to distract too much with the bold or the text. **I also think the buttons should be above the text** since the text only applies to the <10% of people who actually should say no.

Also some light edits/simplification of the text:

```suggestion
    "did_stay_explanation": "Select 'No' if you didn't actually meet. You will not be able to write a public reference, but can optionally share <strong>private</strong> feedback with us about what happened. The other person won’t be informed, but can still leave a reference if they say you met.",
```